src/components/Hero/styles.ts

```diff
@@ -0,0 +1,17 @@
+export const HeroContainer = `
+ flex h-[37.5rem] md:h-[48.25rem]p-[1rem] w-full flex-col items-center md:items-start object-cover bg-center justify-center bg-no-repeat bg-blend-overlay md:p-[9.375rem]
+`
+
+export const TitleContainer = ` 
+flex w-[27.75rem] flex-col items-center text-center md:text-left  justify-center md:items-start md:justify-start 
+`
+
+export const Subtitle = `
+  mb-5 md:w-[26rem] w-[20rem] text-center md:text-2xl text-x1 font-normal text-white md:text-left
```

Comment:
Vamos diminuir o tamanho e mudar a cor para dar um destaque maior para o título
```suggestion
  mb-5 md:w-[26rem] w-[20rem] text-center md:text-xxl text-x1 font-normal text-gray-300 md:text-left
```